CHANGELOG.md

```diff
@@ -2,13 +2,16 @@
 
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.3.8][]
+
+* Add pattern support to `number:list`
```

Comment:
I'd suggest updating this to:

Adding search support to `number:list` using the `pattern` flag